tests/test_spline_fxns.py

```diff
@@ -0,0 +1,326 @@
+import numpy as np
+from brainlit.algorithms.generate_fragments import spline_fxns
+from pytest import approx, raises
+from scipy.interpolate import splprep, splev, BSpline
+
```

Comment:
Can you please demarcate where the input checks are vs where the validation checks are? e.g. line 26,66 of [here](https://github.com/neurodata/brainlit/blob/develop/tests/test_neighborhood.py)

Reply:
Addressed